docs/knowledge-base/DevOps/pipelines.mdx

```diff
@@ -0,0 +1,158 @@
+
+The workflow is building the dokcer image of the service, tagging it and pushing it to `Azure ACR`, along with the docker image it's also pushing the helm package which is part of the repository.  
+Afterwards it's updating the `artifacts.json` which is associated with the team's `scope` under the [helm-charts repository](https://github.com/MapColonies/helm-charts) with the new releases which have been just released.  
+
+### CD part
+
+We're utilizing [Jenkins](https://jenkins.mapcolonies.net/) for this purpose.
+
+#### How to login?
+
+Each team has their own set of credentials which they can use to login to the jenkins instance.
+
+We have 2 options for deployment:
+
+1. Manual deployment
+2. Auto Deploy
+
+#### Auto deployment
+
+The auto deployment process is currently monitoring the following branches: `master` and `qa`. On each commit to the [helm-charts repository](https://github.com/MapColonies/helm-charts) on the monitored branch the pipeline is kicking off.
+
+The pipeline can be found [here](https://jenkins.mapcolonies.net/job/Deploy%20Chart%20Polling%20-%20Multibranch/)  
+As you can see in the overview page of the pipeline there are 2 branches which it's monitoring.
+
+![jenkins auto deploy overview](./img/jenkins-auto-deploy-1.png)  
+
+##### How the auto deploy pipeline is working?
+
+* The pipeline is first of all looking at changes introduced in each commit with a `git diff`.
+* After finding the `diff` it'll look at the `deployment.json` in the `scope` (each team's directory in the `helm-charts` repo) directories.
+* Here's an example of `deployment.json` file
+
+```json
+[
+    {
+      "chartLocation": "3d/charts/ingestion",
+      "valuesFile": ["3d/charts/global-values.yaml", "3d/charts/ingestion/ingestion-values.yaml"],
+      "namespace": "3d-dev",
+      "releaseName": "ingestion-3d",
+      "skip": false
+    },
+    {
+      "chartLocation": "3d/charts/serving",
+      "valuesFile": ["3d/charts/global-values.yaml", "3d/charts/serving/serving-values.yaml"],
+      "namespace": "3d-dev",
+      "releaseName": "serving-3d",
+      "skip": true
+    }
+  ]
+```  
+
+* If you wish to skip a deployment you'll need to have `"skip": true` in the chart you wish to skip.
+* When the pipeline will start it'll send a slack message to the `<team>-notifications` channel based on chart location(s) which are going to be deployed.  
+![jenkins auto deploy slack start message](./img/jenkins-auto-deploy-2.png) 
+* After the pipeline will finish it'll send another message to the `<team>-notifications` channel based on chart location(s) which have been deployed with the status of the deployment.  
+
+## Bundler/Unbundler
+
+The bundler and unbundler workflows are intended to help our dev team to bring in the artifacts which were built in the release flow in an easy way to our internal deployments.
+
+We utilize Jenkins for the bundler and unbundler. 
```

Comment:
We use the Jenkins in all our pipelines, I suggest moving it to the top of the doc